From e745d721e1907e01823c7042688a256efc633e69 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Fri, 2 Mar 2012 03:33:01 +0100
Subject: [PATCH] Revert "vala: avoid potential useless remakes (minor
 bugfix)"

This reverts commit 12dc0ec5e3c7e1a12569d5cfdbb8ceaa5a5f5f58.

Conflicts:

	automake.in
---
 automake.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/automake.in b/automake.in
index 69b6010..4fbe24c 100644
--- a/automake.in
+++ b/automake.in
@@ -6108,7 +6108,7 @@ sub lang_vala_finish_target ($$)
 # move it to its final location only after valac has been invoked.
     "\t${silent}rm -f \$\@ && echo stamp > \$\@-t\n".
     "\t${verbose}\$(am__cd) \$(srcdir) && ${compile} \$(${derived}_SOURCES)\n".
-    "\t${silent}mv -f \$\@-t \$\@\n";
+    "\t${silent}touch \$@\n";
 
   push_dist_common ("${derived}_vala.stamp");
 
-- 
1.7.9.1

